plugins/ag-charts-generate-example-files/src/executors/generate/generator/transformation-scripts/react-utils.ts

```diff
@@ -45,6 +45,7 @@ export function convertTemplate(template: string) {
 }
 
 export function convertFunctionalTemplate(template: string) {
+    if (template.includes('grid-area')) return 'null';
```

Comment:
What does this do here?

Reply:
If there's multiple charts, give up. It wasn't supported earlier, and made syntax errors - and now those syntax errors were now causing the build to fail